File: api/src/main/java/hirez/api/object/statuspage/ScheduledMaintenances.java
package hirez.api.object.statuspage;

import hirez.api.object.interfaces.IStatusPage;
import lombok.Data;

import java.util.List;

@Data
public class ScheduledMaintenances implements IStatusPage {
    private final Page page;
    private final List<ScheduleMaintenance> scheduledMaintenances;
}

File: api/src/main/kotlin/hirez/statuspage/json/ScheduledMaintenances.kt
package hirez.statuspage.json

/**
 *
 * @author Damian Staszewski [damian@stachuofficial.tv]
 * @version %I%, %G%
 * @since 1.0
 */
data class ScheduledMaintenances(
			override val page: StatusPage,
			override val scheduledMaintenances: Array<ScheduleMaintenance>
) : IStatusPage, IScheduledMaintenances